jadx-core/src/test/java/jadx/tests/api/compiler/StaticCompiler.java

```diff
@@ -22,11 +26,17 @@
 
 public class StaticCompiler {
 
+	private static final Logger LOG = LoggerFactory.getLogger(StaticCompiler.class);
+
 	private static final List<String> COMMON_ARGS = Arrays.asList("-source 1.8 -target 1.8".split(" "));
 
 	public static List<File> compile(List<File> files, File outDir, boolean includeDebugInfo) throws IOException {
 
 		JavaCompiler compiler = ToolProvider.getSystemJavaCompiler();
+		if (compiler == null) {
+			LOG.error("Can not find compiler, please use JDK instead");
```

Comment:
Please throw an exception instead of logging, otherwise assert error will raise but with a different reason.